tests/contrib/djangorestframework/app/views.py

```diff
@@ -1,11 +1,19 @@
+import django
 from django.conf.urls import include
-from django.conf.urls import url
 from django.contrib.auth.models import User
 from rest_framework import routers
 from rest_framework import serializers
 from rest_framework import viewsets
 
 
+# django version < 2 does not support django.urls.re_path
+# django version > 3 does not support django.conf.urls.url
```

Comment:
You mean >=4.0?

Reply:
Also, we should be testing both when possible

Reply:
Yup, it's deprecated in django 3 but it's still supported